doc/progress.rst

```diff
@@ -8,6 +8,7 @@ Changelog
 
 0.11.0
 ~~~~~~
+* ADD #929: Fix data edit API
```

Comment:
I don't think this extra line is necessary, as it is a fix for a bug that only ever existed in the `develop` branch and never made it to an official release.